packages/sdk/README.md

```diff
@@ -54,7 +54,7 @@ When the SDK is installed and the server location and API credentials are config
 Verify authentication works and that API calls will succeed with code similar to the following:
 
 ```typescript
-import { LookerNodeSDK } from '@looker/sdk'
+import { LookerNodeSDK } from ''
```

Comment:
We no longer import from the @looker/sdk? Why is this the case now that we have packages?

Reply:
IntelliJ messed up a rename refactor. Thanks, I'll fix it.